tools/update_required_branch_checks.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates the requires status checks for a branch."""
+
+from github import Github
+
+BRANCH_NAME = 'main'
+
+YOUR_TOKEN = ''
+g = Github(YOUR_TOKEN)
+
+repo = g.get_repo('youtube/cobalt')
+latest_open_pr = repo.get_pulls(
```

Comment:
I get what you are doing here, looking at last pull request enabled checks. I thought they had some sort of an API call that just retrieves _all_ existing checks somehow globally as well ? Maybe not though, i thought the `youtube/cobalt/branches/main/protection/required_status_checks/contexts` call would return it but it only returns enabled ones

Reply:
I thought of either
- forcing all checks on LTS branches 
- forcing no checks at all

With forcing all, a situation comes up when the build is broken and you need to get something in, which would require admin powers to bypass. That seems like a workable approach 

Forcing no checks on any branches kind of sort of would work too - but we'll have more breakages from poorly resolved merge conflicts and such.

Reply:
I'm in the Andrew camp re enforcing all checks. IMO the ODT runs are too unstable to rely on for blocking merges.

That said, I'm not planning in staying in this camp for long. If we get over the main issues plaguing us currently we should enforce them.